mirrord-agent/Cargo.toml

```diff
@@ -42,6 +42,7 @@ faccess = "0.2"
 bytes = "1.2"
 regex = "1"
 socket2 = "0.4"
+mockall = "0.11"
```

Comment:
add this as a dev dependency

Reply:
I can't because the `mockall::automock` macro exists on the IPTables struct

Reply:
NVM found how to do it